### DISCUSSION TITLE
Offline boot, raspap won't bring the AP

### DISCUSSION THREAD

**Discussion by PFema** [RaspAP/raspap-webgui#886] [Q&A]
Hello,

I am trying to configure RaspAP as an accesspoint to the RPI. Everytime I boot the raspberry while being connected to my home network it works fine. But as soon as I boot it while being disconnected from my network, the AP won't come up on my other devices. I need to restart all services (raspap, hostapd) in order for itto be available again. 
Another strange thing is that, when offline, the raspap service doesn't seem to want to restart without having to reload the daemon. It seems that something modifies the daemon at boot when the Pi is offline. 
I have seen you guys show a solution to enable raspap in offline mode, but I can't seem to find that solution anymore? 

Any tips?

Also, in offline mode it seems that dhcpcd.conf doesn't override my ethernet static ip address that I set before. While being offline the Pi won't give himself an ethernet IP address at all.

**Comment by billz** [comment]
> as soon as I boot it while being disconnected from my network, the AP won't come up

That's odd. I shutdown my Pi with RaspAP, disconnected eth0, booted and joined the AP no problem. I'm able to browse an attached device with smb over 802.11ac in offline mode. This is with a clean install + default config. 

> I have seen you guys show a solution to enable raspap in offline mode

Don't recall seeing that, at least not one I wrote ;)

**Reply by PFema** [reply]
Hey, I actually solved this, In fact, you're right, there is no problem with offline RaspAP. There is with the Network Manager though. It always brings a lot of problem when offline as it is trying to take over the interfaces somehow (i assumed). Disabling it completely  and only using dhcpcd solved my issues. 
I don't know If that was the best solution.

**Reply by billz** [reply]
This is the solution. NetworkManager is a different animal that does not play nicely with dhcpcd.
